filebeat/input/filestream/config.go

```diff
@@ -23,13 +23,16 @@ import (
 
 	"github.com/dustin/go-humanize"
 
+	loginp "github.com/elastic/beats/v7/filebeat/input/filestream/internal/input-logfile"
 	"github.com/elastic/beats/v7/libbeat/common"
 	"github.com/elastic/beats/v7/libbeat/common/match"
 	"github.com/elastic/beats/v7/libbeat/reader/readfile"
 )
 
 // Config stores the options of a file stream.
 type config struct {
+	ID string `config:"id"`
```

Comment:
Can we document how ID is used?

Is it used for registry state?

Is it used for monitoring?

Reply:
Added documentation.